go.mod

```diff
@@ -33,24 +33,24 @@ require (
 	golang.org/x/tools v0.0.0-20200916195026-c9a70fc28ce3 // indirect
 	gopkg.in/src-d/go-git.v4 v4.13.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.18.8
-	k8s.io/apiextensions-apiserver v0.18.6
-	k8s.io/apimachinery v0.18.8
+	k8s.io/api v0.20.4
+	k8s.io/apiextensions-apiserver v0.20.1
+	k8s.io/apimachinery v0.20.4
```

Comment:
Hmmm this is tricky.. we might lose compatibility with older kube versions. Please, just make sure that we are safe:
https://github.com/kubernetes/client-go#compatibility-matrix

Reply:
I had tested this functionality on k8s v1.20.0. Do I need to test it with some other k8s version?

Reply:
I think we are fine up to 1.18. @sutaakar, the OCP we run our BDD is based on which k8s version?

Reply:
good question, @Kaitou786 could know that

Reply:
Currently we use OCP 4.6.x for our BDD tests which is based on kubernetes 1.19 as stated in the [docs](https://docs.openshift.com/container-platform/4.6/release_notes/ocp-4-6-release-notes.html#ocp-4-6-about-this-release)

Reply:
I had tested these changes on kubernetes 1.19. They are working fine.